msal/src/main/java/com/microsoft/identity/nativeauth/statemachine/states/SignInStates.kt

```diff
@@ -384,15 +385,15 @@ class SignInPasswordRequiredState(
  * SignInAfterSignUpBaseState class is an abstract class to represent signin state after
  * successfull signup
  * in the signin flow.
- * @property signInVerificationCode: Short lived token from signup APIS
+ * @property continuationToken: Continuation token from signup APIS
  * @property username: Username of the user
  * @property config Configuration used by Native Auth
  */
 abstract class SignInAfterSignUpBaseState(
-    internal open val signInVerificationCode: String?,
+    override val continuationToken: String?,
```

Comment:
could you double check that having `internal` in the base class keeps it internal in extending classesI? E.g. in this case, `SignInAfterSignUpBaseState.continuationToken` should not be accessible by the sample app.

Reply:
Tried to access the continuationToken from the sample app and got a compilation error stating it is internal.